cmd/zetaclientd/initconfig.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"cosmossdk.io/errors"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/node/testutil/sample"
+	"github.com/zeta-chain/node/zetaclient/config"
+)
+
+// initializeConfigOptions is a set of CLI options for `init` command.
+type initializeConfigOptions struct {
+	peer               string
+	publicIP           string
+	logFormat          string
+	logSampler         bool
+	preParamsPath      string
+	chainID            string
+	zetacoreURL        string
+	authzGranter       string
+	authzHotkey        string
+	level              int8
+	configUpdateTicker uint64
+
+	p2pDiagnostic       bool
+	p2pDiagnosticTicker uint64
+	TssPath             string
+	TestTssKeysign      bool
```

Comment:
```suggestion
	TSSPath             string
	TestTSSKeysign      bool
```
Let's fix the acronym lowercase when we get the opportunity